keyboards/kbdfans/bella/rgb_iso/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "bella_rgb_iso", 
+    "url": "", 
+    "maintainer": "moyi4681", 
+    "width": 16.25, 
+    "height": 6.25, 
+    "layouts": {
+        "LAYOUT": {
```

May need further changes, but at a minimum
```suggestion
        "LAYOUT_all": {
```